Resources/Prototypes/_NF/Shipyard/ambition.yml

```diff
@@ -0,0 +1,35 @@
+# Author Info
+# GitHub: Cu1r (https://github.com/Cu1r/)
+# Discord: cu1r
+#
+- type: vessel
+  id: Ambition
+  name: UAC Ambition
+  description: Declassified gas production and distribution platform seized in a hostile takeover of an Atmosian conglomerate.
+  price: 140000
+  category: Large
+  group: Civilian
+  shuttlePath: /Maps/_NF/Shuttles/ambition.yml
+
+- type: gameMap
+  id: Ambition
+  mapName: 'UAC Ambition'
+  mapPath: /Maps/_NF/Shuttles/ambition.yml
+  minPlayers: 0
+  stations:
+    Ambition:
+      stationProto: StandardFrontierVessel
```

Comment:
this needs to be StandardFrontierExpeditionVessel in order to allow expeditions for the ship